src/helpers/__tests__/utils.test.js

```diff
@@ -0,0 +1,86 @@
+import { isEmpty } from '../utils';
+
+describe('isEmpty', () => {
+  test('object', () => {
+    {
+      expect(isEmpty({})).toBe(true);
+      expect(isEmpty({ 'a': 1 })).toBe(false);
+    }
+
+    {
+      const a = {};
+      expect(isEmpty(a)).toBe(true);
+      a.a = 1;
+      expect(isEmpty(a)).toBe(false);
+      delete a.a;
+      expect(isEmpty(a)).toBe(true);
+    }
+  });
+
+  test('array', () => {
+    {
+      expect(isEmpty([])).toBe(true);
+      expect(isEmpty([1])).toBe(false);
+    }
+
+    {
+      const a = [];
+      expect(isEmpty(a)).toBe(true);
+      a.push(1);
+      expect(isEmpty(a)).toBe(false);
+      a.pop();
+      expect(isEmpty(a)).toBe(true);
+    }
+  });
+
+  test('null', () => {
+    expect(isEmpty(null)).toBe(true);
+    const a = null;
+    expect(isEmpty(a)).toBe(true);
+  });
+
+  test('undefined', () => {
+    {
+      let a;
+      expect(isEmpty(a)).toBe(true);
+    }
+
+    {
+      const a = { b: '1' };
+      expect(isEmpty(a.b)).toBe(false);
+      expect(isEmpty(a.a)).toBe(true);
+      delete a.b;
+      expect(isEmpty(a.b)).toBe(true);
+    }
+
+    {
+      const a = [1];
+      expect(isEmpty(a[0])).toBe(false);
+      expect(isEmpty(a[1])).toBe(true);
+      a.pop();
+      expect(isEmpty(a[0])).toBe(true);
+    }
+  });
+
+  test('number', () => {
+    expect(isEmpty(1)).toBe(false);
```

Comment:
I would guess any number is empty... 

Why !isNaN is not empty?
What about 0?

Reply:
you think that need add 0 as empty?